readthedocs/v3/models.py

```diff
@@ -0,0 +1,3 @@
+from django.db import models
+
+# Create your models here.
```

Comment:
Same, do we need this file?

Reply:
No. It's created by default when creating a Django app.